examples/udp-client/udp_client.bal

```diff
@@ -2,38 +2,36 @@
 import ballerina/io;
 import ballerina/udp;
 
-public function main() {
+public function main() returns error? {
   
-    // Create a new connectionless udp client.
-    // Optionally, you can provide interface that the socket need to bind 
-    // and the timeout in milliseconds which specifies the read timeout value
-    // udp:Client client = new ("localhost", 2000);
-    udp:Client socketClient = checkpanic new ;
+    // Create a new connectionless UDP client.
+    // Optionally, you can provide the interface that the socket needs to bind 
+    // and the timeout in milliseconds, which specifies the read timeout value.
+    // udp:Client client = new (localHost = "localhost", timeoutInMillis = 2000);
+    udp:Client socketClient = check new;
    
 
     string msg = "Hello Ballerina echo";
     udp:Datagram datagram = {
-        remoteAddress : {
-            host : "localhost",
-            port : 48829
-        },
+        remoteHost: "localhost",
+        remotePort : 48829,
         data : msg.toBytes()
     };
 
-    // Send data to remote host.
-    // The parameter is a Datagram record which contains the remote address
-    // and the data to be.
-    var sendResult = socketClient->send(datagram);
+    // Send the data to remote host.
+    // The parameter is a Datagram record, which contains the `remoteHost`, `remotePort`,
+    // and the `data` to be sent.
+    var sendResult = socketClient->sendDatagram(datagram);
     if (sendResult is ()) {
         io:println("Datagram was sent to the remote host.");
     } else {
         error e = sendResult;
         panic e;
     }
     
-    // Wait until data receive from remote host.
-    var result = socketClient->receive();
-    if (result is udp:Datagram) {
+    // Wait until data is received from the remote host.
+    var result = socketClient->receiveDatagram();
```

Comment:
Let's do a "check" here. And also in the later "close" operation as well.

Reply:
updated